guessingGame.js

```diff
@@ -0,0 +1,34 @@
+pry = require('pryjs')
+
+function guess(number){
```

Comment:
instead of creating these stand-alone functions before creating the `GuessingGame` constructor, it's better practice to define these functions within the `GuessingGame` object or on their prototype so something like this...

```js
function GuessingGame(answer){
  this.answer = answer
  this.isSolved = false
  this.guess = function(number){
	   if(number>this.answer){
	    console.log('high');
	    return 'high';
	  }else if (number < this.answer){
	    console.log('low');
	    return 'low'
	  }else{
	    console.log("correct");
	    this.isSolved = true
	  };
	};
};
```
then, you'll be able to access the functions on the object in a similar way